src/components/screens/SettingsScreen.vue

```diff
@@ -75,35 +133,45 @@ export default {
     }
   },
 
-  data: function() {
+  data: function () {
```

Comment:
any idea why this is adding spaces between function names and parens? seems abnormal

Reply:
Maybe something with my auto formatter for Vue files. I'll look into it.